python/ray/tests/test_advanced.py

```diff
@@ -521,6 +521,43 @@ def method(self):
     assert ray.worker.global_worker.core_worker.object_exists(x_id)
 
 
+@pytest.mark.skipif(client_test_enabled(), reason="internal api")
+def test_future_resolution_skip_plasma(ray_start_cluster):
+    cluster = ray_start_cluster
+    # Disable worker caching so worker leases are not reused; set object
+    # inlining size threshold and enable storing of small objects in in-memory
+    # object store so the borrowed ref is inlined.
+    cluster.add_node(
+        num_cpus=1,
+        resources={"pin_head": 1},
+        _system_config={
+            "worker_lease_timeout_milliseconds": 0,
+            "max_direct_call_object_size": 100 * 1024,
```

Comment:
This test fails if this is set to 0, i.e. when the borrowed object is spilled to Plasma and therefore isn't inlined in the `GetObjectStatus` response.